src/AutoRest.CSharp/Output/Models/Types/SchemaDetection.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using AutoRest.CSharp.Input;
+
+namespace AutoRest.CSharp.Output.Models.Types
```

Comment:
Please move to `Models/Mgmt`

Reply:
this isn't what we will end up with in the end so id rather only move this once  It is scheduled for a future PR https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5725